### ISSUE TITLE
Performance - high CPU load while idling

### ISSUE THREAD

**Issue by anschl** [eclipse-tractusx/tutorial-resources#302]
Hi everyone,

I noticed that the deployed MXD has a high CPU load even though the system is not in use at all:
![image](https://github.com/eclipse-tractusx/tutorial-resources/assets/104145298/562c141f-e6c4-4de7-b6d8-7d0b9ae08744)
![image](https://github.com/eclipse-tractusx/tutorial-resources/assets/104145298/1d261ce8-b46b-4725-a1e8-c56ea65aa404)

Although we are running the MXD on a fairly low powered Server (AWS EC2 t3a.large) I would expect the load to stay low until I request something. We pinned the EDC's version to `0.6.0`.

I find it interesting that the database has the highest load of all pods. Also, Alice's CPU load is higher than Bob's - maybe because it contains more Assets? What else is running on there while idling? Would you say this CPU load is normal?

**Comment by ciprian-nicuta** [comment]
Hi, I deployed mxd with edc 0.6.0 on a t3.large, and from my experience I see a large spike in CPU usage at the initial stage but the usage gets slowly down to an average of 25% after ±20 minutes.

<img width="1413" alt="Screenshot 2024-05-14 at 14 23 04" src="https://github.com/eclipse-tractusx/tutorial-resources/assets/158159743/84c59e5a-c9fc-4a90-8c11-ff2a8dc50a48">

When using Htop we can also see that a half of all cpu usage is dedicated to system and IRQ.
So total user CPU usage after ±20 minutes is around 12% this inludes k8s system pods.

<img width="1173" alt="Screenshot 2024-05-14 at 14 26 54" src="https://github.com/eclipse-tractusx/tutorial-resources/assets/158159743/f07d83fd-6318-412a-b297-6b130553eec6">

Htop CPU colors for reference:
Green: The % of CPU used for user processes 
Red: The % of CPU used by system/kernel processes 
Yellow/Orange: The % of CPU used by IRQ time 

I also checked for pod logs at info level, and there is rarely something happening on default namespace pods.

So I would say this level of CPU usage idling is normal.

How much time after the start the screenshot is taken?
Is CPU usage going down as shown on the first image ?
